target/linux/kirkwood/base-files/etc/diag.sh

```diff
@@ -0,0 +1,58 @@
+#!/bin/sh
```

Comment:
Please drop it. Status led are configured in dts. Please see an example:
https://github.com/openwrt/openwrt/blob/ee1a92834da4dab409dc4cd150bb188c6b069020/target/linux/kirkwood/files/arch/arm/boot/dts/kirkwood-ea3500.dts#L26

Reply:
Removed